pkg/jobs/topology/topology_jobs.go

```diff
@@ -157,3 +157,48 @@ var CleanupDeletedTopologyComponents = &job.Job{
 		return nil
 	},
 }
+
+var TopologyCRDReconcile = &job.Job{
+	Name:       "TopologyCRDReconcile",
+	Schedule:   "@every 1h",
+	Singleton:  true,
+	JobHistory: true,
+	Retention:  job.RetentionBalanced,
+	Fn: func(ctx job.JobRuntime) error {
+		var topologies []models.Topology
+
+		if err := ctx.DB().
+			Select("id", "name", "namespace").
+			Where("source IN (?, ?)", models.SourceCRD, models.SourceTopology).
+			Where(duty.LocalFilter).
+			Find(&topologies).Error; err != nil {
+			return err
+		}
+
+		for _, t := range topologies {
```

Comment:
Should we list all topologies rather than 1 by 1?

Reply:
Done